### PR TITLE
Contact form for the Mobile Portfolio Version

### PR DESCRIPTION
**Pull request by Estherstarr**
- Add contact form for the mobile version portfolio using the form tag and wrapping it in a footer tag.
- Add action to the form using Formspree service
- Style the contact form 
- Add hover and active effects to the button for when different actions are performed on the button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -259,5 +259,18 @@ <h6>Skills</h6>
         </li>
       </ul>
     </section>
+    <footer class="contact-section">
+      <div class="contact-box">
+        <h3 class="h3-contact">Contact me</h3>
+        <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form class="contact-form" action="https://formspree.io/f/mdobgbjy" method="POST">       
+        <input type="text" placeholder="Onomeh"  name="username" required/>
```
Comment (medaminedev66) [inline index.html]: - [ ] As stated in the project requirements: Please ensure that the name input accepts up to 30 chars. You can do this by adding the `maxlength="30"` attribute.

File: style.css [hunk 2]
```diff
@@ -503,3 +503,91 @@ body {
...
+}
+
+.contact-section {
+  background-color: #fff;
+  background-image: url("assets/images/contact-form-shapes-mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-position: top right;
+  width: 380px;
+  height: 779px;
+}
```
Comment (medaminedev66) [inline style.css]: - [ ] Kindly add the background images to the contact section to match the Figma design.
- Your design:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Estherstarr, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️The form is submitted successfully with valid input values. Great job 👍

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
@medaminedev66 Thank you for the review.

I fixed the required changes
-Max length for the input tags
- Background images as shown in the figma design
Kindly review
Thanks for your help

File: index.html [hunk 3]
```diff
@@ -259,5 +259,18 @@ <h6>Skills</h6>
         </li>
       </ul>
     </section>
+    <footer class="contact-section">
+      <div class="contact-box">
+        <h3 class="h3-contact">Contact me</h3>
+        <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form class="contact-form" action="https://formspree.io/f/mdobgbjy" method="POST">       
+        <input type="text" placeholder="Onomeh"  name="username" maxlength="30" required/>
```
Comment (mateo951) [inline index.html:267]: Addressed ✔

File: style.css (hunk 2 above)
Comment (mateo951) [reply style.css]: Addressed 💪

**Review by mateo951 — APPROVED** [review]
### Hi @Estherstarr, 

Well done implementing all the changes required by the previous reviewer 😎
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**